R/parameters.R

```diff
@@ -0,0 +1,208 @@
+#' Read in disease process parameters from an external file or files
+#'
+#' @param generation_interval_path,delay_interval_path,right_truncation_path
+#'   Path to a local file with the parameter PMF. See [read_interval_pmf] for
+#'   details on the file schema. The parameters can be in the same file or a
+#'   different file.
+#' @param disease One of `COVID-19` or `Influenza`
+#' @param as_of_date The date "as of" the run. Set for the current date for the
```

Comment:
Rather than repeat "as of" could say in other words what it means. The date that the run occured?

Reply:
https://github.com/CDCgov/cfa-epinow2-pipeline/pull/16/commits/c144ff64a8ed89067fc94dcf5671b64fd1e841de